### PR TITLE
Port skiplink styling from v3 to v4

### PR DESCRIPTION
**Pull request by patrickhlauke**

### CODE REVIEW

**Comment by patrickhlauke** [comment]
skippy

File: docs/assets/scss/_skiplink.scss [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  padding: 1em;
+  color: #fff;
+  background-color: $bd-purple;
+  outline: 0;
+
+  .skiplink-text {
+    padding: .5em;
+    outline: 1px dotted;
+  }
+}
```
Comment (mdo) [inline docs/assets/scss/_skiplink.scss]: Needs a new line here.

Comment (patrickhlauke) [inline docs/assets/scss/_skiplink.scss]: Done

**Comment by XhmikosR** [comment]
So this should be ready to merge, correct?

**Comment by patrickhlauke** [comment]
Now that @mdo confirmed in slack that there's no shiplist and we can just merge...yes :shipit:

**Pull request merged**